taxonomies/ingredients.txt

```diff
@@ -46060,8 +46127,8 @@ wikipedia:en:https://en.wikipedia.org/wiki/Pea
 fr:extrait de pois
 
 # The chickpea or chick pea (Cicer arietinum) is an annual legume of the family Fabaceae, subfamily Faboideae.
-
-<en:pea
+# chickpea parent is NOT pea
+<en:chickpea
```

Comment:
We cannot have en:chickpea as a parent of itself, we should put something else, like en:legume

Reply:
ok, changed to legume